src/main/resources/api/swagger.yaml

```diff
@@ -225,9 +173,15 @@ definitions:
       author:
         type: string
         description: Name of the feature author
+      status:
+        type: string
+        description: Status of the feature.
       updated_at:
         type: string
         description: Last update date in ISO 8601 format
```

Comment:
I think it's the one preferred on the API Guidelines: http://zalando.github.io/restful-api-guidelines/json-guidelines/JsonGuidelines.html